Vending Machine/VendingMachine/UseCases/BuyUseCase.cs

```diff
@@ -0,0 +1,79 @@
+﻿using iQuest.VendingMachine.DataLayer;
+using iQuest.VendingMachine.PresentationLayer;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace iQuest.VendingMachine.UseCases
+{
+    internal class BuyUseCase : IUseCase
```

Comment:
Hello. Please do not resolve multiple homework on the same PR.
Delete BuyUseCase class, then the PR for LookUseCase will be merged to main.

After that, create a new branch and PR for BuyUseCase homework.